src/global_logic.js

```diff
@@ -1837,10 +1836,9 @@ function getCharaEXLB(chara) {
     Object.keys(EXLB).map((key) => {
         var exactKey = "EXLB" + key;
         if (exactKey in chara) {
-            if(exactKey == "EXLBWED"){
+            if(exactKey == "EXLBWED") {
```

Comment:
- [ ] one more space > "if ("
- [ ] other key check
  - all exactKey has "EXLB" prefix, then can be key == "WED" (or key === "WED")
  - or typedef chara[exactKey] === 'number' for parseInt

Reply:
Object.keys(EXLB).forEach is better here, map() function creates an array, 
the returned value is unused in this code.

This is not change request for PR.
because, it's not only this function, I just note for now.

Reply:
Fixed in ddda901

Reply:
Removed EXLB check in 1594a8f